src/components/exerciseForm/styledInputRange.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+
+const StyledInputRange = styled.input`
+	transform: rotate(-90deg);
+	appearance: none;
+
+	&::-webkit-slider-runnable-track {
+		width: 250px;
+		height: 5px;
+		border-width: thin;
+		border-style: solid;
+		border-radius: 4px;
+		background: #ddd;
+	}
+
+	&::-webkit-slider-thumb {
+		appearance: none;
+		width: 16px;
+		height: 16px;
+		margin-top: -6px;
+		border-style: solid;
+		border-radius: 50%;
+		border-color: white;
+		background: gray;
+	}
```

Comment:
Did you test the appearance in different browsers? 
We highly recommend not to use neither `webkit` nor `appearance: none`, because of the reasons Alex already mentioned. 
By the way an input range slider without a range doesn't make sense 😉

Reply:
i understand the issue, i would like to leave it for now with the styling as it is, it works on chrome firefox and safari but i know with updates of these browsers it can change, so i will change it in the future. But for now i would like to keep it if that is okay?